test/integration/backward_compatible/ClientReconnectAdvancedTest.js

```diff
@@ -77,19 +78,22 @@ describe('ClientReconnectAdvancedTest', function () {
             },
             lifecycleListeners: [ lifecycleListener ]
         });
-        const connectionManager = client.getConnectionManager();
 
-        await assertTrueEventually(async () => {
-            expect(connectionManager.connectionRegistry.activeConnections).to.have.lengthOf(1);
+        await TestUtil.assertTrueEventually(async () => {
+            expect(client.getCluster().getMembers()).to.have.lengthOf(1);
         });
 
         await RC.shutdownMember(cluster.id, member.uuid);
         await disconnectedDeferred.promise;
 
+        await TestUtil.assertTrueEventually(async () => {
+            expect(client.getCluster().getMembers()).to.have.lengthOf(1);
+        });
+
```

Comment:
Just to make test a bit better

Reply:
I guess I need to understand whether member count decreases in case of disconnection. I think member count does not decrease on disconnection. I wonder why we assert member count is 1 three times in this test.

Reply:
Improved the test a little bit. The first member list size check is done to make sure that we are connected. Removed the second check. The third check is replaced by something new that verifies the member list is updated with the new member. Can you check again?

Reply:
We can compare the uuid's as well. Also, testConnectionCountAfterClientReconnect name does not hold anymore, we can just say `testConnectionAfterClientReconnect` or `testConnectionRestoredAfterClientReconnect`

Reply:
done. removed the address and port checks since uuid check should be enough